src/main/java/de/qabel/desktop/ui/LayoutController.java

```diff
@@ -176,10 +176,66 @@ public void initialize(URL location, ResourceBundle resources) {
         newMessageIndicator.visibleProperty().bind(newMessageIndicator.textProperty().isNotEqualTo("0"));
     }
 
+    @FXML
+    BorderPane quotaBlock;
+    @FXML
+    Label quota;
+    @FXML
+    Label provider;
+    @FXML
+    Label quotaDescription;
+    @Inject
+    BoxClient boxClient;
+
+    private void fillQuotaInformation() {
+        try {
+            quotaState = boxClient.getQuotaState();
+        } catch (IOException | QblInvalidCredentials e) {
+            alert(e);
+        }
+        if (quotaState == null) {
+            quotaBlock.setVisible(false);
+            quotaDescription.setVisible(false);
+            return;
+        }
+
+        int ratio = ratioByDiff(quotaState.getSize(), quotaState.getQuota());
+        String quotaDescriptionText = quotaDescription(quotaState.getSize(), quotaState.getQuota());
+        quota.setText(ratio + "%");
+        provider.setMinWidth(ratio);
+        quotaDescription.setText(quotaDescriptionText);
+    }
+
+    int ratioByDiff(long usedQuota, long availableQuota) {
+        return (int) (usedQuota / (double) availableQuota * 100);
+    }
+
+    String quotaDescription(long usedQuota, long availableQuota) {
+        long diff = availableQuota - usedQuota;
+        return getStringSizeLengthFile(diff) + " free / " + getStringSizeLengthFile(availableQuota);
+    }
+
+    private String getStringSizeLengthFile(long size) {
+        DecimalFormat df = new DecimalFormat("0");
+
+        float sizeKb = 1024.0f;
+        float sizeMo = sizeKb * sizeKb;
+        float sizeGB = sizeMo * sizeKb;
+        float sizeTerra = sizeGB * sizeKb;
+
+        if (size < sizeMo)
+            return df.format(size / sizeKb) + " Kb";
+        else if (size < sizeGB)
+            return df.format(size / sizeMo) + " MB";
+        else if (size < sizeTerra)
+            return df.format(size / sizeGB) + " GB";
+
+        return "";
+    }
```

Comment:
Did you copy&paste this code? Then please read it, think about it and ... at least test it